bonita-connector-rest-impl/src/test/java/org/bonitasoft/connectors/rest/AcceptanceTestBase.java

```diff
@@ -70,38 +68,24 @@
     /**
      * The currently used port for the WireMock server
      */
-    private static int port = DEFAULT_PORT;
+    private int port = DEFAULT_PORT;
     
     /**
      * The URL of the WireMock server
      */
-    private static String url = "localhost";

```

Comment:
I think this change should not be part of this Pull Request, as it does not serve the same purpose